### PR TITLE
Refactoring orbital ships

### PR DESCRIPTION
**Pull request by Leibovitch**

### CODE REVIEW

File: drainOnMe/Orbital/Ships/geocalc.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import numpy as np
+
+
+config = json.load(open('drainOnMe/Orbital/Ships/constants.json'))
+ureg = pint.UnitRegistry()
+reference_row = config['reference_row']
+reference_col = config['reference_col']
+step = config['step']
+
+def metric_length(point1, point2):
```
Comment (MasterSplinter334) [inline drainOnMe/Orbital/Ships/geocalc.py]: generally it's bad practice to put numbers in variable names, but i'll let it go here

File: drainOnMe/Orbital/Ships/geocalc.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+    pos1 = [x1 ,y1 ,z1]
+    pos2 = [x2 ,y2, z2]
+
+    distance = config['earth_radius'] * math.acos(np.dot(pos1, pos2)) * ureg.meter
+    return distance.to('kilometer')
+
+def get_coordintes(row, col):
+    delta_row = reference_row[0] - row
+    row_coordinates = reference_row[1] + int(delta_row /3) + (delta_row % 3) *  step
```
Comment (MasterSplinter334) [inline drainOnMe/Orbital/Ships/geocalc.py]: `(delta_row / 3)`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,2 @@
 # drainOnMe
-Draining all the buckets
+downloading analytocs form diferent companies to louvre
```
[the remark this answers is not in the document]
Comment (MasterSplinter334) [reply README.md]: also you said stuff should be in readme about the tiles but it's not here

File: drainOnMe/Orbital/Ships/download_orbital_ships.py [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+                try:
+                    if not Polygon(tile_dictionary[tile_id]).intersects(aoi_polygon):
+                        continue
+                except:
+                    traceback.print_exc()
+                    missing_tiles_log.write(tile_id + '\n')
+                    #this is probably a metadata feature. we continue if the feaure doesn't have a body
+                    continue
+
+                print("reading now file: " + key)
```
[the remark this answers is not in the document]
Comment (Leibovitch) [reply drainOnMe/Orbital/Ships/download_orbital_ships.py]: i ment now. perhaps better phrased as 'now reading'

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Leibovitch) [reply README.md]: i think its a better description. i can change it back if you want

[the remark this answers is not in the document]
Comment (MasterSplinter334) [reply README.md]: no, I meant you have typos, and you said people should read the readme about the tiles...

**Pull request merged**